app/core/utils.js

```diff
@@ -22,6 +22,16 @@ import nurseShark from 'tideline/plugins/nurseshark';
 
 var utils = {};
 
+/**
+ * Convenience function for capitalizing a string
+ * 
+ * @param  {String} str
+ * @return {String}
+ */
+utils.capitalize = (str) => {
+  return str.charAt(0).toUpperCase() + str.slice(1)
+}
+
```

Comment:
Unit test please? There's already a file for these and it'll take like 2sec /test/unit/utils/utils.test.js